architecture/additional_concepts/storage.adoc

```diff
@@ -492,6 +492,183 @@ spec:
         claimName: myclaim
 ----
 
+ifdef::openshift-dedicated[]
+
+[[osd-multi-az-pv]]
+== Multi-AZ persistent volume claims
+
+A walkthrough of common use cases for persistent volumes on a Multi-AZ
+{product-title} cluster.
+
+[[osd-multi-az-pv-config]]
+=== Configuration details
+
+A multi-AZ {product-title} cluster will come configured with one storage class per
+availability zone where the cluster is provisioned. One of these storage
+classes will be set as the default. For example, a cluster provisioned in
+us-east-2 would have three storage classes that are named to indicate their
+availability zone.
+
+.StorageClass availability zone
+[cols="3,2,1",options="header"]
+|===
+|StorageClass Name |Provisioner |Age
+|gp2-encrypted-us-east-2a (default)
+|kubernetes.io/aws-ebs
+|3d
+|gp2-encrypted-us-east-2b
+|kubernetes.io/aws-ebs
+|3d
+|gp2-encrypted-us-east-2c
+|kubernetes.io/aws-ebs
+|3d
+|===
+
+[[osd-multi-az-pv-usecases]]
+=== Use cases
+
+[[osd-multi-az-pv-usecases-pvc-storageclass-cli]]
+==== Creating a PersistentVolumeClaim with a specific storage class (CLI)
+
+Creating a PersistentVolumeClaim (PVC) via the CLI requires a manifest file that
+provides the name and details of the PersistentVolume (PV) being requested.
+To have the PVC use a different storage class, simply indicate that in your
+manifest.
+
+.PVC example
+[source,yaml]
+----
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: mypvc
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+  requests:
+    storage: 1Gi
+  storageClassName: gp2-encrypted-us-east-2b <1>
+----
+<1> Specify your storage class to override the default.
+
+Passing this manifest to `oc create -f <manifest filename>` will create a PV in
+the AZ of the storage class provided. You can confirm this by checking the
+output of `oc get pvc`.
+
+[[osd-multi-az-pv-usecases-pvc-storageclass-ui]]
+==== Creating a PersistentVolumeClaim with a specific storage class (UI)
+
+The web console UI makes choosing a storage class for your PVC easy.
```

Comment:
Remove the first sentence, as we should not indicate what is easy for the user. Consider replacing with:

    To choose a storage class for your PVC using the web console, navigate to the *Storage* dashboard and click 
*Create Storage*.